pkg/controllers/podgroup/pg_controller.go

```diff
@@ -139,6 +142,8 @@ func (pg *pgcontroller) processNextReq() bool {
 	// If no error, forget it.
 	pg.queue.Forget(req)
 
+	go pg.deletePG()
```

Comment:
l think you misunderstood what l mean, l mean it's better to use another worker to delete pg, while not doing it in `processNextReq `

Reply:
so,you hopes i use first version that delete pg in delete eventhandler,but use a gorutine to do it,is that so?

Reply:
@lucming No, l mean you can increase a `deleteWorker` goroutine to do that.
Something like that:
```
go wait.Until(pg.deleteWorker, 0, stopCh)

func (pg *pgcontroller) deleteWorker() {
	for pg.processNextDeleteReq() {}
}

func (pg *pgcontroller) processNextDeleteReq() {
        ...
}
```

Reply:
@shinytang6 done